supporting_content/dimensions/billingperiodstart.md

```diff
@@ -0,0 +1,25 @@
+# Column: BillingPeriodStart
+
+## Example provider mappings
+
+Current column mappings found in available data sets:
+
+| Provider  | Data set                 | Column                      |
+|:----------|:-------------------------|:----------------------------|
+| AWS       | CUR                      | bill/BillingPeriodStartDate |
+| GCP       | BigQuery Billing Export  | invoice.month               |
+| Microsoft | Cost details             | BillingPeriodStartDate      |
+
+## Example usage scenarios
+
+| Provider  | Data set                | Scenario                           | Value                    |
+|:----------|:------------------------|:-----------------------------------|:-------------------------|
+| AWS       | CUR                     | Not available                      | 2023-05-01T00:00:00.000Z |
+| GCP       | BigQuery Billing Export | Not available                      | 202304                   |
+| Microsoft | Cost details            | via Consumption API (usageDetails) | 2022-10-11T00:00:00Z     |
+| Microsoft | Cost details            | via Cost export file               | 02/13/2023               |
+
+## Discussion Topics
+
+* Can multiple invoices be generated for a billing period for a billing account? Yes. E.g. AWS might send multiple invoices for a month for support, usage and adjustments
+* Are billing periods always a month? Can it start from a middle of a month? Yes, commonly done monthly but can be quarterly, yearly etc.
```

Comment:
```suggestion
* Are billing periods always a month? No, commonly done monthly but can be quarterly, yearly etc.
* Can it start from a middle of a month? Yes based on contract terms. This and the fact that it can be longer than a month suggests we need a time period that can be from a point in time to another point in time.
```